README.md

```diff
@@ -116,14 +116,34 @@ val translationConfig = TranslationConfiguration
     .build()
 ```
 
-## Development Setup
-
-### Experimental Languages
-
-Some languages, such as Golang are experimental and depend on other native libraries. Therefore, they are not included in the `cpg-core` module but have separate gradle submodules.
-To include the desired submodules simply toggle them on in your local `gradle.properties` file by setting the value of the properties to `true` e.g., (`enableGoFrontend=true`).
+## Development
+This section describes languages, how well they are supported, and how to use and develop them yourself.
+
+### Language Support
+Languages are maintained to different degrees, and are noted in the table below with:
+- `maintained`: if they are mostly feature complete and bugs have priority of being fixed.
+- `developing`: if the language is currently being worked on to reach a state of feature completeness.
```

Comment:
I think we used the term “incubating”

Reply:
Yes I changed It cause we were not sure what was a good idea or not. I chose this terms due to them being more commonly used, but they might miss a notion that I was not aware we wanted to go for. What do you think, leave it or change `developing` to `incubating` and/or `experimental` to `research`?

Reply:
`experimental` seems fine for me. Personally, I have not heard `developing` for such projects, but `incubating` seems a common term, such as used by the CNCF https://www.cncf.io/projects/, Apache https://incubator.apache.org/projects/, Eclipse https://www.eclipse.org/projects/dev_process/#6_2_3_Incubation